.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1 @@
+distributionUrl=https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.3.3/apache-maven-3.3.3-bin.zip
```

Comment:
this file is only needed when you use these maven-wrappers: https://github.com/takari/maven-wrapper

which I think is good thing to add. we use it for jruby build as well and recently added it to jruby-17 branch